.github/workflows/release-published.yml

```diff
@@ -222,4 +222,46 @@ jobs:
         run: mvn -B clean deploy -DskipTests=true
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          TARGET_ARTIFACT_REPOSITORY: github
\ No newline at end of file
+          TARGET_ARTIFACT_REPOSITORY: github
+
+  deploy_xsd:
+      name: Upload xsds
+      needs: [ setup ]
+      runs-on: ubuntu-20.04
+      outputs:
+          tag: ${{ steps.collect-data.outputs.tag }}
+          version: ${{ needs.setup.outputs.version }}
+      steps:
+      - name: Download liquibase xsd
+        uses: actions/checkout@v3
```

Comment:
Deploying from master rather than what is released risks drift between what's in master and what has been released. 

In theory they should be the same, and I know there was discussion on the implementation options, so I'll just note that we are OK with that risk.

Reply:
@nvoxland What's released will be pushed to master and step to upload XSD's comes right after that

Reply:
we had some discussion around this and as those are xsd's which are not being committed with every push as such. So I still think that would be fine. But if not, I can do something like downloading the entire `.zip` and extracting the `xsd` from there.